app/services/platform-apps/api/modules/streaming-recording.ts

```diff
@@ -51,6 +51,16 @@ export class StreamingRecordingModule extends Module {
     return this.serializeStreamInfo();
   }
 
+  @apiMethod()
+  startStreaming() {
+    this.streamingService.goLive();
+  }
+
```

Comment:
Curious about why this directly goes live instead of toggling streaming?

Reply:
The idea was that since the avatar is managing the stream, it was meant to be a hands free experience that skips the go live window.  This won't work for all platforms so it's definitely something to think about whether this is the right behavior long term.